salt/orchestrate/aws/dogwood_rp_elb.sls

```diff
@@ -8,24 +8,31 @@
 {% endfor %}
 
 {% for edx_type in ['draft', 'live'] %}
+{% set elb_name = 'edx-{0}-dogwood-rp'.format(edx_type) %}
+
 create_elb_for_edx_{{ edx_type }}:
   boto_elb.present:
-    - name: edx-{{ edx_type }}-dogwood-rp
+    - name: {{ elb_name }}
     - listeners:
         - elb_port: 443
           instance_port: 443
           elb_protocol: HTTPS
           instance_protocol: HTTPS
           certificate: arn:aws:iam::610119931565:server-certificate/mitx-wildcard-cert
+          policies:
+            - {{ elb_name }}-sticky-cookie-policy
         - elb_port: 80
           instance_port: 80
           elb_protocol: HTTP
           instance_protocol: HTTP
+          policies:
+            - {{ elb_name }}-sticky-cookie-policy
     - attributes:
         cross_zone_load_balancing:
           enabled: True
-        sticky_sessions:
```

Comment:
No, that was added previously as an attempt to enable them but it doesn't actually do anything. The piece that actually enables them is the `policies` attribute on the listeners. The policy itself is created further down on line 63.